### DISCUSSION TITLE
Forecasting in Categorical Data

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#500] [Q&A] [answered]
Bagaimana cara melaukan forecasting data timeseries untuk target yang berupa categorical ? 
Misalnya saya ingin melakukan prediksi apakah 7 hari kedepan akan hujan atau kemarau.

Terima kasih.

**Comment by kevbow** [comment, marked as answer]
Dear @yohanespm77,

Salah satu syarat dasar dari data time series adalah data targetnya harus bersifat numerikal, oleh karena itu jika kasus yang Anda angkat adalah mencari tahu apakah akan hujan atau tidak maka kasus tersebut bukan lagi time series melainkan klasifikasi. 
Demikian saya sampaikan, semoga membantu.
